spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe 'post_index', type: :feature do
```

Comment:
There are some tests that fail due to some syntax errors. Kindly see the screenshot below and make sure all the tests are passing.
![error](https://user-images.githubusercontent.com/35504149/188711683-ab3a1b38-34d8-463c-9529-96baade77ee7.png)

Reply:
Thanks @SajjadAhmad14 we've looked at your suggested changes.